Readme.md

```diff
@@ -6,20 +6,18 @@
 
 # DevExpress Blazor Components - Set predefined settings for the specific component
 
-Controls usually have the same settings for different pages of a large web application, and we recommend that you specify the component settings in markup. You can also override the **SetParametersAsync** method to specify the default settings for Blazor components in your application. However, we do not recommend that you use this approach because it is not a common approach in Blazor. Use it at your own risk.
+Controls usually have the same settings for different pages of a large web application, and we recommend that you specify the component settings in markup. You can also override the **SetParametersAsyncCore** (for Grid) or **SetParametersAsync** (for other components) method to specify the default settings for Blazor components in your application. However, we do not recommend that you use this technique because it is uncommon in Blazor. Use it at your own risk.
 
 ![Grid with predefined settings and default](images/result.png)
 
-Create an inheritor from our Blazor control that overrides the **SetParametersAsync** method. In this method, set the default values for the public properties before you call the base method. You can initialize properties with custom default values in markup because the base method is called after your custom settings. We do not recommend that you override and change the values of protected properties.
+Inherit from our Blazor control to override the **SetParametersAsync** or **SetParametersAsyncCore** method. In one of these methods, set the default values for the public properties before you call the base **SetParametersXXX** method. In markup, you can initialize properties with modified default values because the base method is called after custom settings are applied. We do not recommend that you override and change the values of protected properties.

```

Comment:
```suggestion
Inherit [inherit what?] from our Blazor control to override the **SetParametersAsync** or **SetParametersAsyncCore** method. In one of these methods [which one? we should be specific - see rephrase below comment], set the default values for the public properties before you call the base **SetParametersXXX** method. In markup, you can initialize properties with modified default values since the base method is called after custom settings are applied. We do not recommend that you override and change the values of protected properties. [ can we tell them why we don't recommend overriding/changing the values of protected properties?]

```
Rewrite. Depending on what is technically right. Since I don't know, I am providing several rewrites.
 You can set the default values for the public properties in either the **SetParametersAsync** or **SetParametersAsyncCore** method before you call the base **SetParametersXXX** method.

Is there a different outcome if they set default values for the public properties in **SetParametersAsync** vs. **SetParametersAsyncCore**?

Reply:
About protected properties: in C#, there are public, protected, and private properties (also applicable for methods). Public properties are accessible at any time via API. Protected properties can be found in the Visual Studio's Watches tab. We do not document protected properties and do not guarantee that we don't change anything in future updates. Also, if someone decides to change anything in our protected members, they should be 100% sure what exactly they are trying to do.
SetParametersAsync vs. SetParametersAsyncCore: the choice depends on the internal implementation of our components. I mention in the previous paragraph what method to use with what component.

```suggestion
Create an inheritor from our Blazor control to override the **SetParametersAsync** or **SetParametersAsyncCore** method. The method override implementation sets the default values for the public properties before you call the base **SetParametersXXX** method. In markup, you can initialize properties with modified default values since the base method is called after custom settings are applied. We do not recommend that you override and change the values of protected properties.

```

Reply:
Note that we do not recommend that you override and change the values of **protected** properties as it may lead to incorrect control behavior.

Reply:
```suggestion
Create an inheritor from our Blazor control to override the **SetParametersAsync** or **SetParametersAsyncCore** method. You can set the default initial values for the **public** properties in either the SetParametersAsync or SetParametersAsyncCore method before you call the base SetParametersXXX method. In markup, you can initialize properties with modified default values since the base method is called after custom settings are applied. 

Note that we do not recommend that you override and change the values of **protected** properties as it may lead to incorrect control behavior.
```